docs/users/SystemRequirements.md

```diff
@@ -31,10 +31,12 @@ This page lists the recommended system requirements to install and run the app o
 <sub>\* Storage space required does not factor in the number of games managed, the size of the managed games and the mods installed.</sub> -->
 
 ### Linux Dependencies
+
 The App requires the following programs to be available at runtime:
```

Comment:
```suggestion
The App requires the following programs to be available on the system `PATH` at runtime:
```

Reply:
I'll add this to my next commit of the user docs @erri120 FYI